draft-ietf-quic-multipath.md

```diff
@@ -444,6 +444,18 @@ a server should issue a token that is capable of validating
 any of the previously validated addresses. Further guidance on token usage can be
 found in {{Section 8.1.3 of QUIC-TRANSPORT}}.
 
+### Immediate path initiation
+
+The server may receive packets for a yet unused Path ID that do not
+contain a path challenge. Such packets are valid if the can be properly decrypted
+and if they contain a valid connection ID. ({{path-initiation}}
+specifies that the client may send data packets immediately if "it has previously
+validated that address".) The server SHOULD create a new path, accept the
+packets arriving on that path from the client, and acknowledge them.
+
+If the server decides to not accept the packet, it MUST close the path as
+specified in {{path-close}}.
```

Comment:
I don't think this really makes sense or actually not even sure what you mean by accepting a packet. You receive a packet from a valid (not-yet used) Path ID; you decrypt it and as we say above this valid not matter what's in there. So either we don't need this sentence at all or maybe you want to say something like if the receiver does not want to open this path, it has to abandon it explicitly.

Reply:
We have another comment about maybe not create a separate section. I can try moving the first paragraph in the main text.